ALVRClient/Renderer.swift

```diff
@@ -98,6 +99,10 @@ class Renderer {
     var fullscreenQuadBuffer:MTLBuffer!
     var lastIpd:Float = -1
     var framesRendered:Int = 0
+    var framesSinceLastIDR:Int = 0
+    var framesSinceLastDecode:Int = 0
+    
+    let constantTimeOffsetForJitterReductionHack:Double = 33.0/1000.0
```

Comment:
Is this constant completely arbitrary or the result of experimentation? Worth adding a comment about

Reply:
I experimented with it, this value seems to be the most stable but I don't intend to keep it around for long